drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -118,13 +121,22 @@ static void nrf5_get_eui64(uint8_t *mac)
 	mac[index++] = IEEE802154_NRF5_VENDOR_OUI & 0xff;
 #endif
 
-#if defined(CONFIG_SOC_NRF5340_CPUAPP) && \
-	defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
+#if defined(CONFIG_SOC_NRF5340_CPUAPP) && defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
 	int ret = -EPERM;
```

Comment:
Is -EPERM a good enough error code for all possible tfm_platform_mem_read error codes?

Reply:
Right, I guess we can map TFM_PLATFORM_ERR_INVALID_PARAM to -EINVAL.

Reply:
Yes. This is the most important error code to provide to the user.

Reply:
Done.